lib/org/chromium/webidl/HTMLFileContents.js

```diff
@@ -16,18 +16,21 @@ foam.CLASS({
       class: 'String',
       name: 'url',
       required: true,
-      final: true,
     },
     {
       class: 'Date',
       name: 'timestamp',
       required: true,
-      final: true,
     },
     {
       class: 'String',
-      name: 'content',
-      final: true,
+      name: 'contents',
+    },
+    {
+      class: 'Array',
+      of: 'String',
```

Comment:
Nits: usually in order: class, of, documentation, name. Please uncomment documentation.